app/views/hyrax/dashboard/collections/new.html.erb

```diff
@@ -1,6 +1,8 @@
 <% provide :page_title, construct_page_title( t('.header', type_title: @collection.collection_type.title) ) %>
+<% a_title = t(:'hyrax.dashboard.collections.new.header') + " " + @collection.collection_type.title %>
```

Comment:
why are you setting this variable here? It looks like you're only using it once. Can you just inline that stuff on line 5?

Reply:
I tried but it that code kept giving me only "New" without the rest of the string. Here's what I tried: <h1><span class="fa fa-edit"></span> <%= t('.header', type_title: t(:'hyrax.dashboard.collections.new.header') + " " + @collection.collection_type.title) %> </h1>

Reply:
Moving this change to the PR: Fix collection action menu edit behavior to go to edit instead of show #2196, because it was blocking travis ci there. If you have a better way, I'm glad to do it.

Reply:
Like so:
```
<%=t(:'hyrax.dashboard.collections.new.header') %> <%= @collection.collection_type.title %> 
```

Reply:
@elrayle - Do you want me to change the header problem on the new page here or should it be a separate issue?